src/main/java/org/sagebionetworks/web/client/widget/modal/Dialog.java

```diff
@@ -32,18 +39,17 @@
 	Button primaryButton;
 	@UiField
 	Button defaultButton;
-	
-	boolean autoHide;
-	
 	@UiField
 	Modal modal;
+	
+	boolean autoHide;
+	Widget widget;
 
 	/**
 	 * Create a new Modal dialog.
 	 */
-	@Inject
-	public Dialog(DialogUiBinder binder) {		
-		initWidget(binder.createAndBindUi(this));
+	public Dialog() {		
```

Comment:
Dialog is merely a form of a container with no dependencies, so a potential philosophy toward these sorts of widgets is that we should use these in the UI XML for clarity, so we don't have:
<Container>
    <DialogContainer/>
</Container>